modelon/impact/client/experiment_definition.py

```diff
@@ -206,6 +216,87 @@ def with_modifiers(self, modifiers=None, **modifiers_kwargs):
             )
         return new
 
+    def with_extensions(self, *experiment_extension):
+        """Sets up an experiment definitions with multiple experiment extensions.
+
+        Parameters:
+
+            experiment_extension --
+                "A variable-length argument of experiment extensions."
+                "Extension object must either be a dictionary or an instance of "
+                "modelon.impact.client.experiment_definition."
+                "SimpleExperimentExtension class."
+
+        Example::
+
+            fmu = model.compile().wait()
+            experiment_definition = fmu.new_experiment_definition(
+                custom_function).with_extensions(SimpleExperimentExtension().
+                with_modifiers({'PI.k': 20}),SimpleExperimentExtension().
+                with_modifiers({'PI.k': 30}))
+            experiment_definition = fmu.new_experiment_definition(
+                custom_function).with_extensions(SimpleExperimentExtension(
+                parameter_modifiers={'start_time': 0.0, 'final_time': 2.0}).
+                with_modifiers({'PI.k': 20}))
+        """
+        exp_ext = []
+        for extension in experiment_extension:
+            if isinstance(extension, SimpleExperimentExtension):
+                exp_ext.extend(extension.to_dict()['extensions'])
```

Comment:
I don't think `to_dict` should be called here. Just store the classes and to_dict can be done when to_dict of SimpleExperimentDefinition is called.